### ISSUE TITLE
Python version of jp.

### ISSUE THREAD

**Issue by bhyde** [jmespath/jp#1]
I wrote a python version since I didn't want to add go to my kit today.

https://gist.github.com/bhyde/6778be7181a5e53af108

Feel free to do with it whatever you like.

**Comment by jamesls** [comment]
Nice.  Also, most of the existing JMESPath libraries offer a script/executable written in their respective language, typically named `jp.<language>`.  So for [python](https://github.com/jmespath/jmespath.py) there's [jp.py](https://github.com/jmespath/jmespath.py/blob/develop/bin/jp.py) as well.

Just for future reference in case any one's curious about why I chose Go for the CLI:

The main reason I went with go for the CLI is because it's a completely self contained executable that's cross platform.  This is nice for windows users as they typically don't have python installed.  It's also nice for linux/mac users as you don't have to install `jmespath` in your global site-packages, which has potential to conflict with other python CLI tools you might have installed globally that also use JMESPath, like the AWS CLI.

As a side benefit, the Go implementation happens to be the fastest JMESpath implementation for the time being ;)
